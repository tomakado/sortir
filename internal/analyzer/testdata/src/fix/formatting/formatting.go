package formatting

var (
	zebra    int
	apple  int // want "variable/constant declarations are not sorted"
	banana        int
)

type MyStruct struct {
	Zebra    string
	Apple  int // want "struct fields are not sorted"
	Banana        bool `json:"banana"`
}