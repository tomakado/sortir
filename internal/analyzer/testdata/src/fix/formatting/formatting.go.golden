package formatting

var (
	apple  int // want "variable/constant declarations are not sorted"
	banana int
	zebra  int
)

type MyStruct struct {
	Apple  int  // want "struct fields are not sorted"
	Banana bool `json:"banana"`
	Zebra  string
}